tox.ini

```diff
@@ -28,17 +26,15 @@ isolated_build = true
 
 [testenv]
 deps =
-    check-manifest >= 0.42
     # If your project uses README.rst, uncomment the following:
     # readme_renderer
     flake8
     pytest
+    build
+    twine
 commands =
-    check-manifest --ignore 'tox.ini,tests/**'
-    # This repository uses a Markdown long_description, so the -r flag to
-    # `setup.py check` is not needed. If your project contains a README.rst,
-    # use `python setup.py check -m -r -s` instead.
-    python setup.py check -m -s
+    python -m build
+    python -m twine check dist/*
```

Comment:
This whole section of the PR is unrelated to moving to `pyproject.toml` (see previous comments to the same effect).

Reply:
Well, there's no `setup.py` to invoke. What would you like to see here?

Reply:
Removing `setup.py` isn't necessary - moving the metadata out is the point of this PR, not removing `setup.py` (or moving off setuptools in general).

Reply:
I'm not sure that I agree with splitting the concomitant configuration changes into separate PRs, but IIUC what you're suggesting is to have a `setup.py` with just:

```py
from setuptools import setup

setup()
```

Is that right?

Reply:
I didn't see any reason to keep an empty `setup.py` around. We're still using `setuptools` as the build backend, so it shouldn't make a difference, right?

Reply:
Agreed, if having `setup.py` isn't needed to keep the functionality the same, then drop it.

Also, if we don't want to include a `data-files` example, and/or an example of using `check-manifest`, by all means drop them. But in separate PRs. If thise PRs get merged before this one, we don't even need to think about how to represent the functionality in `pyproject.toml`.

All I'm asking for here is for each PR to do one well-defined thing, I'm not arguing that we need to keep all the examples we currently have. (TBH, this project already has *way* more in it than I'd prefer - if it were down to me, I'd remove the whole tox configuration, for example).